src/Features/Core/Portable/SpellCheck/AbstractSpellCheckCodeFixProvider.cs

```diff
@@ -29,11 +35,23 @@ internal abstract class AbstractSpellCheckCodeFixProvider<TSimpleName> : CodeFix
 
             var syntaxRoot = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             var node = syntaxRoot.FindNode(span);
```

Comment:
>FindNode [](start = 34, length = 8)

I didn't follow the "node or token" logic. When would we not find a node?

Reply:
@jcouv when you have a ConstructorDeclaration, the 'name' is not an IdentifierNameSyntax, it's just an 'Identifier Token'.